src/hub.html

```diff
@@ -46,6 +46,7 @@
     >
         <a-assets>
             <img id="rotation-symbol" crossorigin="anonymous" src="./assets/rotation.png">
+            <img id="button" crossorigin="anonymous" src="./assets/hud/button.9.png">
             <img id="action-button" crossorigin="anonymous" src="./assets/hud/action_button.9.png">
```

Comment:
Weird, was this just broken before? Seems like this PR is adding the image but the reference was already there.

Reply:
no, this is a new image and new reference I added to make the buttons more pink

Reply:
actually specifically i updated `action_button.9.png` to be pink, and now the `button` mixin references this `img` which is the old, grey button background.